packages/autorest.python/autorest/codegen/serializers/builder_serializer.py

```diff
@@ -1138,8 +1169,9 @@ def handle_error_response(self, builder: OperationType) -> List[str]:
                     f"        {async_await} response.read()  # Load the body in memory and close the socket",
                 ]
             )
+        type_ignore = "  # type: ignore" if builder.has_special_status_code else ""
         retval.append(
-            "    map_error(status_code=response.status_code, response=response, error_map=error_map)"
+            f"    map_error(status_code=response.status_code, response=response, error_map=error_map){type_ignore}"
```

Comment:
To fix mypy error about `error_map` of azure-core>=1.29.5: https://dev.azure.com/azure-sdk/public/_build/results?buildId=3345438&view=logs&j=e21df8c4-2408-587b-0ef3-88cd7352cfd7&t=4052b419-4367-5687-9afd-fcfd51354992&l=47